front/src/api.js

```diff
@@ -0,0 +1,11 @@
+export const fetchData = async () => {
+    try {
+        const response = await fetch('http://127.0.0.1:8000/api/employees/');
+        const employees = await response.json();
+        console.log("fetchData");
+        console.log(employees);
+        return employees;
+    } catch (e) {
+        console.log(e);
```

Comment:
Let's remove all these `console.log` statements. And for the errors, you could think on handling them in some nice way, i.e. add a store part to store errors (usually it would be inside the related data domain, i.e. `employees` part in the store can have `errors` field to it). Then you can display these errors to the user in a nice way.